docusaurus/docs/protocol/actors/supplier.md

```diff
@@ -25,16 +25,32 @@ Configurations to stake an `Supplier` can be found at [supplier_staking_config.m
 
 ## CLI
 
+The `Supplier` actor depends on both the [`supplier`](https://github.com/pokt-network/poktroll/tree/main/x/supplier)
+and [`proof`](https://github.com/pokt-network/poktroll/tree/main/x/proof) on-chain modules.
+These two modules' concerns are seperated as follows:
+
+### Supplier Module
+- Supplier (un/)staking
+- Supplier querying
+
+### Proof Module
+- Claim creation & querying
+- Proof submission & querying
+
 All of the read (i.e. query) based operations for the `Supplier` actor can be
-viewed by running:
+viewed by running the following:
 
 ```bash
 poktrolld query supplier
+# or
+poktrolld query proof
 ```
```

Comment:
Can you split these so it's like this so the user would just click on the RHS to copy-paste:

```bash
poktrolld query supplier
```
or
```
poktrolld query proof
```